stable/nfs-server-provisioner/README.md

```diff
@@ -75,7 +75,7 @@ their default values.
 | `storageClass.name`            | The name to assign the created StorageClass                                                                     | `nfs`                                                    |
 | `storageClass.allowVolumeExpansion` | Allow base storage PCV to be dynamically resizeable (set to null to disable )                              | `true                                                    |
 | `storageClass.parameters`      | Parameters for StorageClass                                                                                     | `{}`                                                     |
-| `storageClass.mountOptions`    | Mount options for StorageClass                                                                                  | `[ "vers=4.1", "noatime" ]`                              |
+| `storageClass.mountOptions`    | Mount options for StorageClass                                                                                  | `[ "vers=3", "relatime" ]`                               |
```

Comment:
What will the effects be of changing the defaults for users already running the chart? Will it break when trying to go from nfs v 4.1 to nfs 3? If so perhaps it would be best to leave `4.1` as the default and have the user override this. Furthermore if we simply omit `noamtime` will that result in `relatime` since it is the default if I'm reading https://linux.die.net/man/8/mount correctly

Reply:
Hi @naseemkullah mountOptions will not be affected for already created volumes, only for new ones.

> Furthermore if we simply omit noamtime will that result in relatime since it is the default if I'm reading https://linux.die.net/man/8/mount correctly

This is good question, I'm not sure, so we need to find out that.

Reply:
All right `relatime` is enabled by default:

```
# mount.nfs -V
mount.nfs: (linux nfs-utils 1.3.3)
```